frog/imports/api/activities.js

```diff
@@ -118,15 +129,20 @@ export const copyOperatorIntoGraphOperator = (
   });
 };
 
-export const removeGraph = graphId => Meteor.call('graph.flush.all', graphId);
+export const removeGraph = (graphId: string) =>
+  Meteor.call('graph.flush.all', graphId);
 
 export const deleteDatabase = () => Meteor.call('graph.flush.db');
 
-export const dragGraphActivity = (id, xPosition) => {
+export const dragGraphActivity = (id: string, xPosition: number) => {
   Activities.update(id, { $inc: { xPosition } });
 };
 
-export const addOperator = (operatorType, data = {}, id) => {
+export const addOperator = (
+  operatorType: string,
+  data: Object = {},
+  id: string
+) => {
   if (id) {
```

Comment:
Either you should make id optional in the type, or you should remove the if (id) check :) Seems like we do two different things based on the existence of id, though, so you should make it optional?